### PR TITLE
Fix skp_generator for flutter_lints 2.0

### PR DESCRIPTION
**Pull request by goderbauer**
See https://github.com/flutter/flutter/pull/99881

### CODE REVIEW

**Comment by goderbauer** [comment]
Those failures look all very unrelated... looking into them...

**Review by dnfield — APPROVED** [review]
LGTM provided CI is happy

File: scripts/verify_tests_on_master.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ flutter/bin/flutter doctor -v
 export PATH="$PATH":`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin
 
 cd flutter/dev/customer_testing
-pub get
+dart pub get
```
Comment (dnfield) [inline scripts/verify_tests_on_master.sh:32]: Can you make the same change in verify_tests_on_master.bat?

Comment (goderbauer) [reply scripts/verify_tests_on_master.sh:32]: Done!

**Review by dnfield — APPROVED** [review]

**Pull request merged**